src/client/setup.py

```diff
@@ -42,18 +44,15 @@ def load_conf():
     args['library_dirs'] = [os.path.join(conf['PREFIX'], 'lib64')]
     args['runtime_library_dirs'] = args['library_dirs']
 
-if sys.version_info < (3, 0):
-    shim_name = 'pydaos_shim_27'
-else:
-    shim_name = 'pydaos_shim_3'
-    args['define_macros'] = [('__USE_PYTHON3__', 1)]
+shim_name = 'pydaos_shim_3'
```

Comment:
You can probably get rid of the shim_name variable and just use the string directly in the module1 = line below.